### DISCUSSION TITLE
Security Contact Inquiry

### DISCUSSION THREAD

**Discussion by tenbbughunters** [fluent/fluent-bit#8799] [Q&A]
Hi all,

We've attempted to contact this repositories maintainers through the contacts listed on the security policy page (https://github.com/fluent/fluent-bit/security/policy) as well as opened a vulnerability report through the repositories mechanism but haven't received a response.

Is there a dedicated contact anyone knows of that we can get in touch with?

Thanks

**Comment by patrick-stephens** [comment]
Can you try again? It's just a Google Group and I am a member but have not seen anything come through.
